Tests/PointFreeTests/__Snapshots__/StripeWebhooksTests/testNoInvoiceSubscriptionId.1.Conn.txt

```diff
@@ -4,14 +4,10 @@ Stripe-Signature: t=1517356800,v1=bda3ac3f25b3665eac23aa47f7d521d7cb4578a70cbebf
 
 {"id":"evt_test","data":{"object":{"id":"in_test","amount_paid":1700,"customer":"cus_test","lines":{"has_more":false,"data":[{"amount":1700,"id":"ii_test","quantity":1,"subscription":"sub_test","plan":{"amount":1700,"name":"Individual Monthly","id":"monthly-2019","nickname":"Individual Monthly","created":1517356800,"tiers":[{"amount":1600},{"amount":1800}],"currency":"usd","metadata":{},"interval":"month"}}]},"charge":"ch_test","total":1700,"period_start":1517356800,"subtotal":1700,"amount_remaining":0,"date":1517356800,"period_end":1519948800,"number":"0000000-0000"}},"type":"invoice.payment_failed"}
 
-400 Bad Request
-Content-Length: 52
-Content-Type: text/plain
+200 OK
```

Comment:
this is now correct. the request can now be a successful 200 because it can get the subscription id out of the invoice line items.